packages/components/src/Tabs/Tabs.css

```diff
@@ -47,35 +51,43 @@ ul.tabRow {
 
 .tabRow li {
   display: flex;
+  position: relative;
   list-style: none;
 }
 
 .tab {
-  flex: 0 0 auto;
   height: var(--tab--height);
   margin: 0;
-  padding-left: var(--public-tab--inset);
-  padding-right: var(--public-tab--inset);
-  border: solid var(--border-base) var(--color-border);
-  border-left-width: 0;
+  padding: var(--space-smaller) var(--space-small);
```

Comment:
Updated padding from SG-1.
TODO: Determine if the `public-tab-inset` needs to be updated for the content

Reply:
Regressions were fixed in https://github.com/GetJobber/atlantis/pull/1899/commits/fcd477b82174c76fdff8398931eaf360ab57bb19